Modules/getpath.py

```diff
@@ -363,10 +363,20 @@ def search_up(prefix, *landmarks, test=isfile):
         venv_prefix = None
         pyvenvcfg = []
 
+    # Search for the 'home' key in pyvenv.cfg. Currently, we don't consider the
+    # presence of a pyvenv.cfg file without a 'home' key to signify the
+    # existence of a virtual environment — we quietly ignore them.
+    # XXX: If we don't find a 'home' key, we don't look for another pyvenv.cfg!
```

Comment:
This comment is a bit confusing - there is only one searched path for `pyvenv.cfg`. Are you suggesting that there should be multiple places being looked at for `pyvenv.cfg`?

Reply:
Sorry for the delay getting back to you.

No, there are two paths, `{executable_dir}` and `{executable_dir}/..`. Using the `venv` install scheme, `pyvenv.cfg` will be at `{executable_dir}/..`, but implementation also allows it to be alongside the executable.

See https://docs.python.org/3.14/c-api/init_config.html#init-path-config

> If [home](https://docs.python.org/3.14/c-api/init_config.html#c.PyConfig.home) is not set and a pyvenv.cfg file is present in the same directory as [executable](https://docs.python.org/3.14/c-api/init_config.html#c.PyConfig.executable), or its parent, (...)